lib/acts_as_tenant/controller_extensions.rb

```diff
@@ -13,7 +13,7 @@ def set_current_tenant_by_subdomain(tenant = :account, column = :subdomain )
       self.tenant_column = column.to_sym
 
       self.class_eval do
-        before_filter :find_tenant_by_subdomain
+        before_action :find_tenant_by_subdomain
```

Comment:
To keep backwards compatibility with < Rails 4, you can do this:

```rb
if respond_to?(:before_action)
  before_action :find_tenant_by_subdomain
else
  before_filter :find_tenant_by_subdomain
end
```

This seems to be the accepted method from what I've gathered, checking others trying for the same.